tests/tt_eager/python_api_testing/unit_testing/misc/test_moreh_arange.py

```diff
@@ -200,12 +195,7 @@ def test_arange_tilized_major_optioanl_output(start_end_step, optional_output, d
     tt_dev = tt_npu.cpu().to_torch()
 
     tt_dev = (
-        tt_npu.cpu()
-        .to(ttl.tensor.Layout.ROW_MAJOR)
-        .unpad_from_tile((1, 1, 1, L))
-        .to_torch()
-        .reshape((L))
-        .to(torch.bfloat16)
+        tt_npu.cpu().to(ttl.tensor.Layout.ROW_MAJOR).unpad_from_tile((1, L)).to_torch().reshape((L)).to(torch.bfloat16)
```

Comment:
Do pad_to_tile and unpad_from_tile still not support 1-dimensional Tensors?

Reply:
Yes, still not support 1D tensor.
```Python
tt_npu.cpu().to(ttl.tensor.Layout.ROW_MAJOR).unpad_from_tile((1, L)).
```
if i try this code, runtime error occur. because output tensor is 2D.

Therefore, if an attempt is made to create a 1D tensor, it results in an error accordingly.

Reply:
> Yes, still not support 1D tensor.
> 
> ```python
> tt_npu.cpu().to(ttl.tensor.Layout.ROW_MAJOR).unpad_from_tile((1, L)).
> ```
> 
> if i try this code, runtime error occur. because output tensor is 2D.
> 
> Therefore, if an attempt is made to create a 1D tensor, it results in an error accordingly.

Do you mean this?
```tt_npu.cpu().to(ttl.tensor.Layout.ROW_MAJOR).unpad_from_tile((L))```

Reply:
> Yes, still not support 1D tensor.
> 
> ```python
> tt_npu.cpu().to(ttl.tensor.Layout.ROW_MAJOR).unpad_from_tile((1, L)).
> ```
> 
> if i try this code, runtime error occur. because output tensor is 2D.
> 
> Therefore, if an attempt is made to create a 1D tensor, it results in an error accordingly.


Is this impossible?
```cpp
        output = (
            ttl.tensor.Tensor(output_cpu, ttl.tensor.DataType.BFLOAT16)
            .reshape([1, L])
            .pad_to_tile(float("nan"))
            .to(ttl.tensor.Layout.TILE)
            .to(device)
        )
```

Reply:
```Python
    start = 0
    end = 10
    step = 1
    tt_cpu = torch.arange(start=start, end=end, step=step).to(torch.bfloat16)
    output_cpu = torch.empty_like(tt_cpu)

    # this work
    output = (
            ttl.tensor.Tensor(  tensor=output_cpu,
                                data_type=ttl.tensor.DataType.BFLOAT16, 
                                device=device, 
                                layout=ttl.tensor.Layout.ROW_MAJOR,
                                mem_config=ttl.tensor.MemoryConfig(ttl.tensor.TensorMemoryLayout.INTERLEAVED, ttl.tensor.BufferType.DRAM)
                                )
        )

   # this segmentation fault
    output = (
            ttl.tensor.Tensor(  tensor=output_cpu,
                                data_type=ttl.tensor.DataType.BFLOAT16, 
                                device=device, 
                                layout=ttl.tensor.Layout.TILE,
                                mem_config=ttl.tensor.MemoryConfig(ttl.tensor.TensorMemoryLayout.INTERLEAVED, ttl.tensor.BufferType.DRAM)
                                )
        )
```

Yes. The reason it is possible to create a 1D tensor is that it is created in a `row-major` order.
It seems that creating a 1D tensor does not work in a tilized layout.

Reply:
We are working on uplifting Tensor shape / layout, it will be awesome 

Your use case will be supported

fyi @nsmithtt